flink-libraries/flink-ml/src/main/scala/org/apache/flink/ml/optimization/GradientDescent.scala

```diff
@@ -272,7 +272,7 @@ abstract class GradientDescent extends IterativeSolver {
   * The regularization function is `1/2 ||w||_2^2` with `w` being the weight vector.
   */
 class GradientDescentL2 extends GradientDescent {
-
+  //TODO(skavulya): Pass regularization penalty as a parameter
```

Comment:
I've mentioned this in the previous PR but adding here for completeness:

I'm in favor of adding the regularization penalty as a parameter for the optimizer.

However that would involve changes that perhaps beyond the scope of this PR, 
currently with only SGD available we don't have to worry about the applicability of L1/L2 regularization, but should add a note for when L-BFGS get implemented. 

Depending on how much work @skavulya it would be to make the change here, we can choose to have a separate PR for that or include it here.